### PR TITLE
program to return lists of odd and even integers

### PR DESCRIPTION
**Pull request by JMSudhan**

### CODE REVIEW

**Review by lnarasim — CHANGES_REQUESTED** [review]
Check spelling

**Review by lnarasim — COMMENTED** [review]
Check spelling

File: pyproblems/factorial_number.py [hunk 1]
```diff
@@ -0,0 +1,14 @@
+''' This program returns the factorial of the number passes'''
```
Comment (lnarasim) [inline pyproblems/factorial_number.py]: Check spelling

File: pyproblems/factorial_number.py [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+from pyproblems.utility import is_int
+def factorial_num(number):
+    '''The function will return the factorial of the given number'''
+    if not is_int(number):
+        raise TypeError(f'unsupported format, {type(number)} is in the place of"integer"')
+    if number < 0:
+        raise ValueError("pass a positive integer")
+    if number == 1 or 0:
```
Comment (lnarasim) [inline pyproblems/factorial_number.py]: Is this condition correct?

File: pyproblems/factorial_number.py [hunk 3]
```diff
@@ -0,0 +1,14 @@
+''' This program returns the factorial of the number passes'''
+from pyproblems.utility import is_int
+def factorial_num(number):
```
Comment (lnarasim) [inline pyproblems/factorial_number.py:3]: have you tested this code for 0!

File: pyproblems/sep_odd_eve.py [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+        if not is_int(i):
+            raise TypeError("unsupported format, pass integers inside the list")
+    odd_list = []
+    even_list = []
+    for i in list_int:
+        if i % 2 != 0:
+            odd_list.append(i)
+        else:
+            even_list.append(i)
+    return [odd_list, even_list]
```
Comment (lnarasim) [inline pyproblems/sep_odd_eve.py]: can you return as tuple instead?

**Review by lnarasim — CHANGES_REQUESTED** [review]
Please check the comments and resubmit the review

File: pyproblems/factorial_number.py (hunk 3 above)
Comment (JMSudhan) [reply pyproblems/factorial_number.py:3]: Yes It has a test case for 0 in the test_factorial.py file

File: pyproblems/factorial_number.py (hunk 2 above)
Comment (JMSudhan) [reply pyproblems/factorial_number.py]: Yes sir, the condition is correct and it yields the desired output.

File: pyproblems/factorial_number.py (hunk 3 above)
Comment (lnarasim) [reply pyproblems/factorial_number.py:3]: Check the spelling of "arguement". It is should be "argument".

File: pyproblems/factorial_number.py (hunk 2 above)
Comment (lnarasim) [reply pyproblems/factorial_number.py]: for the above condition. whether the "if" block execute if number is zero?

**Review by lnarasim — CHANGES_REQUESTED** [review]
Please check the comments again. Resubmit

**Review by lnarasim — APPROVED** [review]

**Pull request merged**